### ISSUE TITLE
Can I use  MMdetection framework Version 2.x?

### ISSUE THREAD

**Issue by huwei1024** [DevashishPrasad/CascadeTabNet#137]
Can I use  MMdetection framework Version 2.x?

**Comment by mfl22** [comment]
Yes, you need to upgrade the pretrained model using the `Tools/upgrade_model_version.py` script and probably modify the configuration file `Config/cascade_mask_rcnn_hrnetv2p_w32_20e_v2.py` slightly (read the error messages, if any, and simply copy relevant parameters from `mmdetection/configs`).
